addons/binding/org.openhab.binding.s7/src/main/java/org/openhab/binding/s7/handler/S7BaseThingHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.openhab.binding.s7.handler;
+
```

Comment:
I also meant this class was looking empty, imho an interface would maybe more appropriate?